webapp/components/new-test-results-history-grid.js

```diff
@@ -166,45 +167,48 @@ class TestResultsGrid extends PathInfo(PolymerElement) {
     this.chartRunIDs[chartIndex] = [];
 
     // Create a row for each subtest
-    this.subtestNames.forEach(subtestName => {
-      if (!browserTestData[subtestName]) {
-        return;
-      }
-      for (let i = 0; i < browserTestData[subtestName].length; i++) {
-        const dataPoint = browserTestData[subtestName][i];
-        const startDate = new Date(dataPoint.date);
-
-        // Use the next entry as the end date, or use present time if this
-        // is the last entry
-        let endDate = now;
-        if (i + 1 !== browserTestData[subtestName].length) {
-          const nextDataPoint = browserTestData[subtestName][i + 1];
-          endDate = new Date(nextDataPoint.date);
-        }
+    if (this.subtestNames) {
```

Comment:
Maybe we should check this in the beginning of the function `if (!browserTestData) || !this.subtestNames)`